### PR TITLE
Update Vagrant

### PR DESCRIPTION
**Pull request by fbernhart**
vagrant

**Issue:** #3169 
**Alexa rank:** -

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json` --> (wasn't necessary)
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I took the already used SVG.
I used Inkscape to fix size and center
I used https://jakearchibald.github.io/svgomg/ to optimize the resulting SVG

### CODE REVIEW

File: icons/vagrant.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Vagrant icon</title><path d="M5.672 6.774V4.917l3.164-1.822L3.56.005.397 1.852v2.263L7.52 21.41 12 23.995v-6.496l2.107-1.224-.024-.015 4.245-9.486V4.917l5.275-3.065L20.439.005l-5.272 3.087h-.003V5.202L12 12.584v2.467l-2.11 1.224zm3.164-3.66L8.814 3.1 5.672 4.917v1.857l4.218 9.501L12 15.234v-2.65L8.836 5.202zm9.492 1.803v1.857l-4.22 9.101L12 17.332v6.663l4.521-2.607L23.603 4.05V1.852z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Vagrant icon</title><path d="M5.6694 6.7718V4.914l3.1653-1.8227L3.5565 0 .3922 1.8478v2.264l7.126 17.3021L12 24v-6.4987l2.1079-1.2245-.024-.015 4.2467-9.49V4.914l5.2772-3.0662L20.4425 0l-5.2742 3.0883h-.003v2.1109L12 12.5842v2.468l-2.1109 1.2246zm3.1653-3.6615l-.022-.014L5.6694 4.914v1.8578l4.2197 9.505L12 15.2353v-2.651L8.8347 5.1991zm9.496 1.8037v1.8578l-4.2218 9.1048L12 17.3342V24l4.5229-2.608 7.085-17.3453v-2.199z"/></svg>
```
Comment (adamrusted) [inline icons/vagrant.svg:1]: Perfect! 💯
This seems to be 5 decimals in precision, does it reduce down to 3?

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/vagrant.svg (hunk 1 above)
Comment (fbernhart) [reply icons/vagrant.svg]: Done 👍🏼

File: icons/vagrant.svg [hunk 2]
```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Vagrant icon</title><path d="M5.672 6.774V4.917l3.164-1.822L3.56.005.397 1.852v2.263L7.52 21.41 12 23.995v-6.496l2.107-1.224-.024-.015 4.245-9.486V4.917l5.275-3.065L20.439.005l-5.272 3.087h-.003V5.202L12 12.584v2.467l-2.11 1.224zm3.164-3.66L8.814 3.1 5.672 4.917v1.857l4.218 9.501L12 15.234v-2.65L8.836 5.202zm9.492 1.803v1.857l-4.22 9.101L12 17.332v6.663l4.521-2.607L23.603 4.05V1.852z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Vagrant icon</title><path d="M5.67 6.772V4.914l3.165-1.823L3.556 0 .392 1.848v2.264l7.126 17.302L12 24v-6.499l2.108-1.224-.024-.015 4.247-9.49V4.914l5.277-3.066L20.443 0l-5.275 3.088h-.003V5.2L12 12.584v2.468l-2.11 1.225zM8.834 3.11l-.022-.014-3.144 1.818v1.858l4.22 9.505L12 15.235v-2.65L8.835 5.198zm9.496 1.804v1.858l-4.222 9.105L12 17.334V24l4.523-2.608 7.085-17.345v-2.2z"/></svg>
```
Comment (adamrusted) [inline icons/vagrant.svg:1]: Looks like there's some unnecessary paths in this one - could you look at removing them?
image

I've included my path for reference:
```svg
M23.608,1.845L20.44,0l-5.274,3.087V3.09v2.107l-3.168,7.385L8.834,5.197V3.109L8.817,3.1l0.017-0.01L3.556,0L0.392,1.845 V4.11l7.123,17.3L11.998,24l4.523-2.611l7.083-17.344V1.847L23.608,1.845z
```

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/vagrant.svg (hunk 2 above)
Comment (fbernhart) [reply icons/vagrant.svg]: I've now rebuilt it from source (`Vagrant_PrimaryLogo_Black_RGB.svg` from the media kit) and removed the unnecessary paths. Please have another look at it. 😊

**Review by adamrusted — APPROVED** [review]
LGTM! 💪🏻

**Pull request merged**